test/extended/util/test.go

```diff
@@ -533,6 +533,9 @@ var (
 			`\[sig-network\] Services should have session affinity work for service with type clusterIP`,
 			// SDN-587: OVN-Kubernetes doesn't support hairpin services
 			`\[sig-network\] Services should allow pods to hairpin back to themselves through services`,
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1763936
+			`\[sig-network\] Networking Granular Checks: Services should function for node-Service`,
+			`\[sig-network\] Networking Granular Checks: Services should function for pod-Service`,
```

Comment:
This is in the `[Skipped:Network/OVNKubernetes]` section, which means it will only be skipped for clusters using ovn-kubernetes, which is not what you mean. You presumably want to skip regardless of network plugin, so this would go in the main skip list.

Reply:
Thank you, that's valid information finally!